proprietary/tokens/src/components/ams/blockquote.tokens.json

```diff
@@ -6,7 +6,7 @@
       "font-size": { "value": "{ams.text.level.3.font-size}" },
       "font-weight": { "value": "{ams.text.font-weight.bold}" },
       "line-height": { "value": "{ams.text.level.3.line-height}" },
-      "inverse-color": { "value": "{ams.color.text.inverse}" }
+      "color-inverse": { "value": "{ams.color.text.inverse}" }
```

Comment:
This must be nested. ‘inverse’ is the modifier, ‘color’ the CSS property. So:

```json
  "inverse": {
    "color": { "value": "{ams.color.text.inverse}" }
  }
```

And likewise for other components.